frontend/src/scenes/notebooks/Nodes/NotebookNodeFlagCodeExample.tsx

```diff
@@ -0,0 +1,47 @@
+import { createPostHogWidgetNode } from 'scenes/notebooks/Nodes/NodeWrapper'
+import { NotebookNodeType } from '~/types'
+import { useValues } from 'kea'
+import { FeatureFlagLogicProps, featureFlagLogic } from 'scenes/feature-flags/featureFlagLogic'
+import { FeatureFlagCodeExample } from 'scenes/feature-flags/FeatureFlagCodeExample'
+import { urls } from 'scenes/urls'
+import { JSONContent, NotebookNodeViewProps } from '../Notebook/utils'
+import { notebookNodeLogic } from './notebookNodeLogic'
+
+const Component = (props: NotebookNodeViewProps<NotebookNodeFlagCodeExampleAttributes>): JSX.Element => {
+    const { id } = props.node.attrs
+    const { featureFlag } = useValues(featureFlagLogic({ id }))
+    const { expanded } = useValues(notebookNodeLogic)
+
+    return <div className="p-2">{expanded && <FeatureFlagCodeExample featureFlag={featureFlag} />}</div>
+}
+
+type NotebookNodeFlagCodeExampleAttributes = {
+    id: FeatureFlagLogicProps['id']
+    expandedOnLoad?: boolean
```

Comment:
I believe we already have the behaviour you're looking for in the `startExpanded` prop. It can be passed as an option to the `createPostHogWidgetNode` method and shouldn't need to be an attribute on the node

Reply:
Ahhh, nice!

Reply:
interesting, curious - do you want the expanded analytic event to be sent for startExapnded?

Reply:
Mmh good point - probably not. I think the intention there is that it only fires when someone intentionally opens the node